pha-admin/src/main/java/com/store/pharmacy/common/serviceImpl/LabelServiceImpl.java

```diff
@@ -21,14 +21,14 @@
     LabelRepository labelRepository;
 
     @Override
-    public List<HashMap> findByLabelCodes(List<LabelInput> labelInputs){
+    public List<HashMap> findByLabelCodes(List<LabelInput> labelInputs, String lang){
         List<HashMap> outputParams = new ArrayList<>();
         if(labelInputs.isEmpty()){
             throw new DataNotFoundException();
         }
         for (LabelInput input: labelInputs) {
             HashMap labelOutput = new HashMap();
```

Comment:
Provide the parametrized type for this generic. For example: List<String> myList;